src/main/scala/ai/privado/languageEngine/java/tagger/sink/JavaAPITagger.scala

```diff
@@ -109,6 +109,7 @@ class JavaAPITagger(cpg: Cpg, ruleCache: RuleCache, privadoInputConfig: PrivadoI
   override def runOnPart(builder: DiffGraphBuilder, ruleInfo: RuleInfo): Unit = {
     val apiInternalSources = cpg.literal.code("(?:\"|')(" + ruleInfo.combinedRulePattern + ")(?:\"|')").l
     val propertySources    = cpg.property.filter(p => p.value matches (ruleInfo.combinedRulePattern)).usedAt.l
+    val serviceSource      = cpg.property.filter(p => p.value matches ".*(http|https):\\/\\/.*").usedAt.l
```

Comment:
can we update the combinedRulePattern itself instead.

Reply:
yes sure, I need to raise the PR for rule change also.

Reply:
hard coded in code itself as of now. 
CC: @HiteshMah-Jan